### ISSUE TITLE
PDF viewer language doesn't match users nextcloud language

### ISSUE THREAD

**Issue by mjanssens** [nextcloud/files_pdfviewer#78]
### Steps to reproduce
1. select and view a random pdf file inside Nextcloud webpage
2. look at language displayed for pdf viewer zoom controls, sidebar, tooltips.

### Expected behaviour
All items should follow the language setting configured by the user, for me that would be Dutch

### Actual behaviour
All items inside pdf viewer are displayed in English language

## Server configuration detail

**Operating system:** Linux 4.13.16-2-pve #1 SMP PVE 4.13.16-47 (Mon, 9 Apr 2018 09:58:12 +0200) x86_64

**Webserver:** Apache (apache2handler)

**Database:** pgsql PostgreSQL 9.6.7 on x86_64-pc-linux-gnu, compiled by gcc (Debian 6.3.0-18) 6.3.0 20170516, 64-bit

**PHP version:** 7.0.27-0+deb9u1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, apache2handler, PDO, xml, apcu, apc, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, imagick, intl, json, exif, mcrypt, pdo_pgsql, pgsql, Phar, posix, readline, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, Zend OPcache

**Nextcloud version:** 13.0.2 - 13.0.2.1

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - activity: 2.6.1
 - admin_audit: 1.3.0
 - announcementcenter: 3.2.1
 - apporder: 0.4.1
 - bookmarks: 0.11.0
 - bruteforcesettings: 1.0.3
 - calendar: 1.6.1
 - caniupdate: 0.1.2
 - checksum: 0.4.0
 - comments: 1.3.0
 - contacts: 2.1.3
 - dav: 1.4.6
 - deck: 0.3.1
 - federatedfilesharing: 1.3.1
 - federation: 1.3.0
 - files: 1.8.0
 - files_accesscontrol: 1.3.0
 - files_downloadactivity: 1.2.0
 - files_external: 1.4.1
 - files_markdown: 2.0.4
 - files_pdfviewer: 1.2.1
 - files_sharing: 1.5.0
 - files_texteditor: 2.5.1
 - files_trashbin: 1.3.0
 - files_versions: 1.6.0
 - files_videoplayer: 1.2.0
 - firstrunwizard: 2.2.1
 - gallery: 18.0.0
 - gpxedit: 0.0.8
 - gpxmotion: 0.0.6
 - gpxpod: 2.2.2
 - groupfolders: 1.2.0
 - impersonate: 1.0.4
 - issuetemplate: 0.3.0
 - limit_login_to_ip: 1.0.4
 - logreader: 2.0.0
 - lookup_server_connector: 1.1.0
 - mail: 0.7.10
 - news: 12.0.1
 - nextcloud_announcements: 1.2.0
 - notes: 2.3.2
 - notifications: 2.1.2
 - oauth2: 1.1.0
 - password_policy: 1.3.0
 - phonetrack: 0.2.7
 - polls: 0.8.1
 - previewgenerator: 1.0.9
 - provisioning_api: 1.3.0
 - quota_warning: 1.2.0
 - ransomware_protection: 1.1.0
 - richdocuments: 2.0.8
 - serverinfo: 1.3.0
 - sharebymail: 1.3.0
 - spreed: 3.1.0
 - survey_client: 1.1.0
 - systemtags: 1.3.0
 - tasks: 0.9.6
 - theming: 1.4.1
 - twofactor_backupcodes: 1.2.3
 - twofactor_totp: 1.4.1
 - updatenotification: 1.3.0
 - workflowengine: 1.3.0
Disabled:
 - encryption
 - user_external
 - user_ldap

```
</details>

## Client configuration

**Browser:** Mozilla/5.0 (X11; Linux x86_64; rv:59.0) Gecko/20100101 Firefox/59.0

**Comment by mjanssens** [comment]
This appears to be fixed in PDF Viewer 1.3.2 in Nextcloud 14 beta 3

**Comment by mjanssens** [comment]
issue remains in 13.0.6, but still fixed in 14RC2, so closing
